README.md

```diff
@@ -231,6 +168,8 @@ The IBM Spectrum Scale Container Storage Interface (CSI) driver has the followin
 
 2. Update `deploy/spectrum-scale-driver.conf` with your cluster and environment details.
```

Comment:
should we refer operator way for installing plugin ?  Do we need to put installer script in our readme , since we want user to use operator for installation ?

Reply:
Have added operator method in README